### PR TITLE
Fixed KubeCon NA Banner mobile layout

### LINKED ISSUE #43622: Newly added KubeCon banner had some issues
**This is a Bug Report**

**Problem:**
1. There is some background color difference of banner, when its viewed-on laptop and when its viewed-on mobile. Shade of colour can vary a little depending upon device's display, but here this difference is a much.
2. There is a little text wrapping issue too in banner.

Screenshot (826)
Screenshot_2023-10-21-07-19-07-246_com android chrome

### PR DESCRIPTION
**Pull request by Gauravpadam**
Closes #43622

This fix should work for other announcements as well

### CODE REVIEW

**Review by aj11anuj — COMMENTED** [review]
As per deploy preview, it looks good to me 👍

File: data/announcements/scheduled.yaml [hunk 1]
```diff
@@ -203,7 +203,7 @@ announcements:
     color: #fffff;
   title: |
     <img src ="/images/announcements/kccnc-na-2023-white.svg" style="float: right; height: 80px;" />
-    <a href= "https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/">KubeCon + CloudNativeCon NA 2023</a> <em> Chicago, Illinois + Virtual</em>.
+    <a href= "https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/">KubeCon + CloudNativeCon NA 2023</a> <em> Chicago, Illinois + Virtual</em>
```
Comment (sftim) [inline data/announcements/scheduled.yaml:206]: Why this change?

File: assets/scss/_custom.scss [hunk 2]
```diff
@@ -806,28 +806,31 @@ body.td-documentation {
...
-        float: initial;
+      a {
+        font-weight: bold;
       }
+      
+      row-gap: 1rem;
+      font-size: 1.5em;
+
+      img {
+        order: 2;
```
Comment (sftim) [inline assets/scss/_custom.scss:823]: How do we know this value is correct?

File: assets/scss/_custom.scss [hunk 3]
```diff
@@ -806,28 +806,31 @@ body.td-documentation {
...
-    .announcement-main, aside .announcement-main {
-      padding-top: calc(min(2rem,2em));
-    }
-  }
-}
+    padding-bottom: 0;
+    h4{
+      display: flex;
+      flex-direction: column;
+      text-align: center;
```
Comment (sftim) [inline assets/scss/_custom.scss:813]: How do we know this is right (for every future announcement)?

Comment (Gauravpadam) [reply assets/scss/_custom.scss:813]: column flex would work well with small viewports, I'd like to know your thoughts  on this

File: assets/scss/_custom.scss (hunk 2 above)
Comment (Gauravpadam) [reply assets/scss/_custom.scss:823]: I see the problem here, I'll figure something out

File: data/announcements/scheduled.yaml (hunk 1 above)
Comment (Gauravpadam) [reply data/announcements/scheduled.yaml:206]: That's uncalled for, I'll revert this

Comment (sftim) [reply data/announcements/scheduled.yaml:206]: Please do update this PR; ideally, also rebase it against _main_.

**Comment by reylejano** [comment]
Hi @Gauravpadam , since this PR has needed a rebase for more than a few months, I'm closing this PR for now.
When you have to rebase this PR, you can reopen this PR with `/reopen` in a comment
/close
